.buildkite/scripts/steps/functional/scalability_dataset_extraction.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+USER_FROM_VAULT="$(retry 5 5 vault read -field=username secret/kibana-issues/dev/apm_parser_performance)"
+PASS_FROM_VAULT="$(retry 5 5 vault read -field=password secret/kibana-issues/dev/apm_parser_performance)"
+ES_SERVER_URL="https://kibana-ops-e2e-perf.es.us-central1.gcp.cloud.es.io:9243"
+JOB_ID=${BUILDKITE_JOB_ID}
+
+.buildkite/scripts/bootstrap.sh
+
+journeys=("login" "ecommerce_dashboard" "flight_dashboard" "web_logs_dashboard" "promotion_tracking_dashboard" "many_fields_discover")
+
+for i in "${journeys[@]}"; do
+    JOURNEY_NAME="${i}"
+    echo "Looking for JOURNEY ${JOURNEY_NAME} and JOB_ID ${JOB_ID} in APM traces"
+
+    yarn cli -u "${USER_FROM_VAULT}" -p "${PASS_FROM_VAULT}" -c "${ES_SERVER_URL}" -s "2022-04-12T00:01:00.000Z" -j "${JOB_ID}" -n "${JOURNEY_NAME}"
```

Comment:
I'm not sure if `yarn cli` will be a good name inside kibana, maybe we might want to change npm script that is more clear we are using **@elastic/performance-testing-dataset-extractor**

Reply:
I agree, I put it rather as command template. I'm thinking to create a script file under `scripts/` or smth we can run with reference to Kibana repo.

Reply:
@suchcodemuchwow adressed in b8db2216d0a21bc10a102a3196fec5001288956c